omnigibson/__init__.py

```diff
@@ -72,6 +73,38 @@ def print_save_usd_warning(_):
 def create_app():
     global app
     from omni.isaac.kit import SimulationApp
+    app = SimulationApp({"headless": gm.HEADLESS or bool(gm.REMOTE_STREAMING)})
+
+    # Default Livestream settings
+    if gm.REMOTE_STREAMING:
+        app.set_setting("/app/window/drawMouse", True)
+        app.set_setting("/app/livestream/proto", "ws")
+        app.set_setting("/app/livestream/websocket/framerate_limit", 120)
+        app.set_setting("/ngx/enabled", False)
+
+        from omni.isaac.core.utils.extensions import enable_extension
+
+        hostname = socket.gethostname()
+
+        # Note: Only one livestream extension can be enabled at a time
+        if gm.REMOTE_STREAMING == "native":
+            # Enable Native Livestream extension
+            # Default App: Streaming Client from the Omniverse Launcher
+            enable_extension("omni.kit.livestream.native")
+            print(f"Now streaming on {hostname} via Omniverse Streaming Client\n")
```

Comment:
super nit: why "\n" in the end

Reply:
Great question. I guess it must have been to leave space after this but I think it's OK to remove.